src/Microsoft.Crank.Models/Microsoft.Crank.Models.csproj

```diff
@@ -2,12 +2,12 @@
 
   <PropertyGroup>
     <Description>Class to transfer data to/from the benchmark server.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetVersion)" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="3.1.8" />
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Should we instead change the SDK to use the aspnet one so it gets this ref for free?

Reply:
It asks for a Main method at that point.

Reply:
I think you should be able to set `OutputType` to `Library` and that should fix this, but anyway, just a NIT, so feel free to disregard.